internal/p10k.zsh

```diff
@@ -4332,7 +4332,7 @@ prompt_gcloud() {
   _p9k_read_word ~/.config/gcloud/active_config || return
   if ! _p9k_cache_stat_get $0 ~/.config/gcloud/configurations/config_$_p9k__ret; then
     # TODO: Use `gcloud config configurations list` instead.
-    _p9k_cache_stat_set "$(gcloud config get-value account 2>/dev/null)" "$(gcloud config get-value project 2>/dev/null)"
+    _p9k_cache_stat_set "$(gcloud config get-value account 2>/dev/null)" "$(gcloud projects describe `gcloud config get-value project 2>/dev/null` | grep name | cut -d ":" -f 2 2>/dev/null)"
```

Comment:
Not sure it matters, but maybe `awk '{print $2}'` instead of `cut -d ":" -f 2` to get rid of the leading whitespace?

```
$ echo "-$(gcloud projects describe `gcloud config get-value project 2>/dev/null` | grep name | cut -d ":" -f 2 2>/dev/null)-"
- testproject-
```

```
$ echo "-$(gcloud projects describe `gcloud config get-value project 2>/dev/null` | grep name | awk '{print $2}' 2>/dev/null)-"
-testproject-
```

Reply:
The real code in powerlevel10k doesn't have this line. Project name is extracted with the following command:

```zsh
gcloud projects describe $P9K_GCLOUD_PROJECT_ID --configuration=$P9K_GCLOUD_CONFIGURATION \
    --account=$P9K_GCLOUD_ACCOUNT --format='value(name)'
````